Products/CMFPlone/controlpanel/browser/overview.py

```diff
@@ -62,11 +63,11 @@ def server_info(self):
             try:
                 server = pkg_resources.get_distribution(server_name)
                 server_version = server.version
-            except (
-                pkg_resources.DistributionNotFound,
-                pkg_resources.RequirementParseError,
-            ):
-                pass
+            except Exception:
+                __import__("pdb").set_trace()
+                warnings.warn(
+                    f"Cannot find or parse version for {self.request.get('SERVER_SOFTWARE')}"
```

Comment:
Feel free to ignore: I think it is good to have it like this (this will use the `__repr__` of whatever `self.request.get('SERVER_SOFTWARE'` is)
```suggestion
                    "Cannot find or parse version for %r", self.request.get('SERVER_SOFTWARE')
```